<?xml version="1.0"?>
<launch>
    <!-- arguments -->
    <arg name="pcd_map" default="outdoor.pcd" />
    <arg name="points_topic" default="/map_points" />
    <arg name="rate" default="0.1" />

    <!-- pcd to poincloud conversion -->
    <node pkg="pcl_ros" type="pcd_to_pointcloud" name="pcd_map" args="$(find mbs_husky_slam)/maps/$(arg pcd_map) $(arg rate)">
        <remap from="/cloud_pcd" to="$(arg points_topic)" />
    </node>
</launch>
